backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationMutationResolver.java

```diff
@@ -131,7 +131,6 @@ public Organization createOrganization(String name, String externalId, String te
             String orderingProviderCity, String orderingProviderCounty, String orderingProviderState,
             String orderingProviderZipCode, String orderingProviderTelephone, List<String> deviceIds,
             String defaultDeviceId) {
-        _os.assertFacilityNameAvailable(testingFacilityName);
```

Comment:
This get the users current organization. Making it impossible to add an organization when you arn't in an organization

Reply:
I would advocate removing the method entirely, and doing a better job of wrapping the error we get from the unique key constraint violation, but that's not a today solution. 😄